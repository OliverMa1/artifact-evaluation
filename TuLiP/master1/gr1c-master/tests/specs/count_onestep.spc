# Expected output after synthesis and calling list_aut_dump() is in
# tests/expected_outputs/count_onestep.spc.listdump.out

ENV: ;
SYS: y [0,2];
ENVINIT: ;
ENVTRANS:;
ENVGOAL: ;

SYSINIT: ;
SYSTRANS: ;

# The subformula y != 0 is included as a regression test for erroneous
# != operator expansion (cf. https://github.com/tulip-control/gr1c/issues/17)
SYSGOAL: []<>(y = 1 & y != 0);
