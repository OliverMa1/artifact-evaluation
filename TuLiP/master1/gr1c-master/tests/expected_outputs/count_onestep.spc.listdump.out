   0 : 1 -  0 -  0 - [ 0]
   1 (init) : 0 -  0 -  1 - [ 0]
